apps/builder/app/builder/features/style-panel/shared/color-picker.tsx

```diff
@@ -1,19 +1,33 @@
 import { useState } from "react";
-import { colord, type RgbaColor } from "colord";
+import { colord, extend, RgbaColor } from "colord";
+// eslint-disable-next-line import/no-internal-modules
+import namesPlugin from "colord/plugins/names";
```

Comment:
I'd put it as an exception in eslint rule. Practically all modern packages with "exports" in package.json forbid to import internal paths when it's not part of api.